### PR TITLE
Added loading spinners to dashboard

### LINKED ISSUE #1849: Repository Dashboard needs loading spinners
See this example - I used Chrome and emulated a slow 3G connection

### PR DESCRIPTION
**Pull request by maufcost**
I re-used obo's spinner from `obojobo-document-engine` for consistency. Let me know if you would like me to re-design another one exclusively for the dashboard. The spinner is centered horizontally as requested and has a subtle margin from the top. As always, I am open for any changes.

Fixes #1849

### CODE REVIEW

**Review by zachberry — CHANGES_REQUESTED** [review]
This is working great! I'd like to add the same spinner to when modules are deleted as well (both bulk delete and deleting one module)

**Review by jpeterson976 — DISMISSED** [review]
LGTM! Spinners are working for both creating and deleting modules. Good work!

**Comment by jpeterson976** [comment]
Also, this seems to cover #1853 as well

File: packages/app/obojobo-repository/shared/components/dashboard.scss [hunk 1]
```diff
@@ -28,4 +28,11 @@
 			}
 		}
 	}
+
+	.test {
```
Comment (zachberry) [inline packages/app/obojobo-repository/shared/components/dashboard.scss:32]: Left in by mistake?

**Review by zachberry — CHANGES_REQUESTED** [review]
Looks great! Just have one code question for you

File: packages/app/obojobo-repository/shared/components/dashboard.scss (hunk 1 above)
Comment (maufcost) [reply packages/app/obojobo-repository/shared/components/dashboard.scss:32]: Oh Christ. Yeah, I'm gonna remove that.

**Review by zachberry — APPROVED** [review]

**Review by jpeterson976 — APPROVED** [review]

**Pull request merged**